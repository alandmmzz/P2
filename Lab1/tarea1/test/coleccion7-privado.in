crearColeccion

crearFecha 01/09/2005
crearLibro 999001122 "Nunca me abandones" "Kazuo" "Ishiguro" "Jóvenes clonados enfrentan su destino en un mundo distópico" 2
agregarEnColeccion

crearFecha 08/06/1949
crearLibro 123456789 "1984" "George" "Orwell" "Un régimen totalitario controla hasta los pensamientos de sus ciudadanos" 2
agregarEnColeccion

crearFecha 29/07/1954
crearLibro 181920212 "El señor de los anillos: La comunidad del anillo" "J.R.R." "Tolkien" "Una épica aventura para destruir un anillo de poder" 6
agregarEnColeccion

crearFecha 01/01/1869
crearLibro 101112131 "Guerra y paz" "León" "Tolstói" "Épica historia sobre la nobleza rusa durante las guerras napoleónicas" 3
agregarEnColeccion

crearFecha 01/01/2006
crearLibro 141516171 "Cometas en el cielo" "Khaled" "Hosseini" "Una amistad marcada por la traición en el Afganistán contemporáneo" 1
agregarEnColeccion

crearFecha 19/10/1953
crearLibro 151617181 "Fahrenheit 451" "Ray" "Bradbury" "Un bombero quema libros en una sociedad donde leer es delito" 2
agregarEnColeccion

crearFecha 01/08/1965
crearLibro 161718192 "Dune" "Frank" "Herbert" "Una lucha política y espiritual en el desierto del planeta Arrakis" 5
agregarEnColeccion

crearFecha 17/09/1951
crearLibro 194512348 "Bestiario" "Julio" "Cortázar" "Siete cuentos extraños y perturbadores que exploran lo fantástico en lo cotidiano" 1
agregarEnColeccion

crearFecha 01/01/1952
crearLibro 121314151 "El viejo y el mar" "Ernest" "Hemingway" "Un viejo pescador enfrenta al mar en una lucha silenciosa" 1
agregarEnColeccion

crearFecha 01/01/1989
crearLibro 224512789 "El palacio de la luna" "Paul" "Auster" "Un joven cae en la pobreza y encuentra sentido en un viaje inesperado" 1
agregarEnColeccion

crearFecha 01/01/2007
crearLibro 234567891 "El informe de Brodeck" "Philippe" "Claudel" "Un hombre escribe un informe sobre un crimen en un pueblo marcado por la posguerra" 1
agregarEnColeccion

imprimirColeccion

removerDeColeccion 4
removerDeColeccion 7
removerDeColeccion 10

imprimirColeccion

imprimirLibrosGeneroColeccion 2

liberarColeccion

Fin