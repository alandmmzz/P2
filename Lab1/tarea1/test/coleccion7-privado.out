1>2>3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>18>19>20>21>22>23>24>25>26>27>28>29>30>31>32>33>34>35>Libro 101112131
Título: Guerra y paz
Autor: León Tolstói
Descripción: Épica historia sobre la nobleza rusa durante las guerras napoleónicas
Género: 3
Fecha de edición: 1/1/1869
Libro 123456789
Título: 1984
Autor: George Orwell
Descripción: Un régimen totalitario controla hasta los pensamientos de sus ciudadanos
Género: 2
Fecha de edición: 8/6/1949
Libro 194512348
Título: Bestiario
Autor: Julio Cortázar
Descripción: Siete cuentos extraños y perturbadores que exploran lo fantástico en lo cotidiano
Género: 1
Fecha de edición: 17/9/1951
Libro 121314151
Título: El viejo y el mar
Autor: Ernest Hemingway
Descripción: Un viejo pescador enfrenta al mar en una lucha silenciosa
Género: 1
Fecha de edición: 1/1/1952
Libro 151617181
Título: Fahrenheit 451
Autor: Ray Bradbury
Descripción: Un bombero quema libros en una sociedad donde leer es delito
Género: 2
Fecha de edición: 19/10/1953
Libro 181920212
Título: El señor de los anillos: La comunidad del anillo
Autor: J.R.R. Tolkien
Descripción: Una épica aventura para destruir un anillo de poder
Género: 6
Fecha de edición: 29/7/1954
Libro 161718192
Título: Dune
Autor: Frank Herbert
Descripción: Una lucha política y espiritual en el desierto del planeta Arrakis
Género: 5
Fecha de edición: 1/8/1965
Libro 224512789
Título: El palacio de la luna
Autor: Paul Auster
Descripción: Un joven cae en la pobreza y encuentra sentido en un viaje inesperado
Género: 1
Fecha de edición: 1/1/1989
Libro 999001122
Título: Nunca me abandones
Autor: Kazuo Ishiguro
Descripción: Jóvenes clonados enfrentan su destino en un mundo distópico
Género: 2
Fecha de edición: 1/9/2005
Libro 141516171
Título: Cometas en el cielo
Autor: Khaled Hosseini
Descripción: Una amistad marcada por la traición en el Afganistán contemporáneo
Género: 1
Fecha de edición: 1/1/2006
Libro 234567891
Título: El informe de Brodeck
Autor: Philippe Claudel
Descripción: Un hombre escribe un informe sobre un crimen en un pueblo marcado por la posguerra
Género: 1
Fecha de edición: 1/1/2007
36>No se puede remover el libro con isbn 4 porque NO pertenece a la colección.
37>No se puede remover el libro con isbn 7 porque NO pertenece a la colección.
38>No se puede remover el libro con isbn 10 porque NO pertenece a la colección.
39>Libro 101112131
Título: Guerra y paz
Autor: León Tolstói
Descripción: Épica historia sobre la nobleza rusa durante las guerras napoleónicas
Género: 3
Fecha de edición: 1/1/1869
Libro 123456789
Título: 1984
Autor: George Orwell
Descripción: Un régimen totalitario controla hasta los pensamientos de sus ciudadanos
Género: 2
Fecha de edición: 8/6/1949
Libro 194512348
Título: Bestiario
Autor: Julio Cortázar
Descripción: Siete cuentos extraños y perturbadores que exploran lo fantástico en lo cotidiano
Género: 1
Fecha de edición: 17/9/1951
Libro 121314151
Título: El viejo y el mar
Autor: Ernest Hemingway
Descripción: Un viejo pescador enfrenta al mar en una lucha silenciosa
Género: 1
Fecha de edición: 1/1/1952
Libro 151617181
Título: Fahrenheit 451
Autor: Ray Bradbury
Descripción: Un bombero quema libros en una sociedad donde leer es delito
Género: 2
Fecha de edición: 19/10/1953
Libro 181920212
Título: El señor de los anillos: La comunidad del anillo
Autor: J.R.R. Tolkien
Descripción: Una épica aventura para destruir un anillo de poder
Género: 6
Fecha de edición: 29/7/1954
Libro 161718192
Título: Dune
Autor: Frank Herbert
Descripción: Una lucha política y espiritual en el desierto del planeta Arrakis
Género: 5
Fecha de edición: 1/8/1965
Libro 224512789
Título: El palacio de la luna
Autor: Paul Auster
Descripción: Un joven cae en la pobreza y encuentra sentido en un viaje inesperado
Género: 1
Fecha de edición: 1/1/1989
Libro 999001122
Título: Nunca me abandones
Autor: Kazuo Ishiguro
Descripción: Jóvenes clonados enfrentan su destino en un mundo distópico
Género: 2
Fecha de edición: 1/9/2005
Libro 141516171
Título: Cometas en el cielo
Autor: Khaled Hosseini
Descripción: Una amistad marcada por la traición en el Afganistán contemporáneo
Género: 1
Fecha de edición: 1/1/2006
Libro 234567891
Título: El informe de Brodeck
Autor: Philippe Claudel
Descripción: Un hombre escribe un informe sobre un crimen en un pueblo marcado por la posguerra
Género: 1
Fecha de edición: 1/1/2007
40>Libro 123456789
Título: 1984
Autor: George Orwell
Descripción: Un régimen totalitario controla hasta los pensamientos de sus ciudadanos
Género: 2
Fecha de edición: 8/6/1949
Libro 151617181
Título: Fahrenheit 451
Autor: Ray Bradbury
Descripción: Un bombero quema libros en una sociedad donde leer es delito
Género: 2
Fecha de edición: 19/10/1953
Libro 999001122
Título: Nunca me abandones
Autor: Kazuo Ishiguro
Descripción: Jóvenes clonados enfrentan su destino en un mundo distópico
Género: 2
Fecha de edición: 1/9/2005
41>42>Fin.
